### DISCUSSION TITLE
BIPES Translations

### DISCUSSION THREAD

**Discussion by rafaelaroca** [BIPES/BIPES#92] [Announcements]
Hello,

We have pushed to GitHub **partial** translations to:
Deutsch, Spanish, Chinese, Italian, and French.

The commit is this one:
https://github.com/BIPES/BIPES/commit/83708985d7ad59a5bc1a22306b7696b0bbcca793

All the translations are here:
https://github.com/BIPES/BIPES/tree/master/ui/msg

The files on the mentioned dir contain the translations for each language. We have left the English message for some words. Help would be appreciated to translate these files completely. If you want to help, you just have to edit one of these files, translate the words and make a git push with the fully translated file!

**Comment by wilderMann** [comment]
I would love to help you translate a bit more stuff to german, unfortunately I cannot find sensors like OneWire, or actuators like RC Servo and loads of other things, neither in the en.js, nor de.js or pt-br.js.

Can I find the keywords somewhere to add it to the de.js and provide translation?
